gymnasium/envs/mujoco/__init__.py

```diff
@@ -1,2 +1,2 @@
-from gymnasium.envs.mujoco.mujoco_env import MujocoEnv, MuJocoPyEnv
```

Comment:
We do not want to export `BaseMujocoEnv`

Reply:
Done